src/main/java/com/gtnewhorizons/angelica/mixins/Mixins.java

```diff
@@ -50,6 +49,11 @@ public enum Mixins {
             ,"sodium.MixinTessellator"
             ,"sodium.MixinWorldClient"
             ,"sodium.MixinGuiIngameForge"
+            ,"sodium.MixinEffectRenderer"
+            // TODO Doesn't work correctly
```

Comment:
Are you planning to address these now, or should I merge now and deal with them later?

Reply:
In this PR, I've laid the groundwork for this optimization. It's theoretically feasible to implement it, but I haven't been able to do so yet. I believe I might be able to create another PR later, or you might be able to complete the code